src/components/SettingsAdvanced.vue

```diff
@@ -0,0 +1,230 @@
+                    <i v-else class="fa fa-times" aria-hidden="true" @click="filterString = ''"/>
+                </div>
+            </div>
+            <table class="u-table kiwi-settings-advanced-table" cellspacing="0">
+                <p v-if="filteredSettings.length === 0" class="kiwi-settings-advanced-noresult">
+                    {{ $t('settings_advanced_noresult') }}
+                </p>
+                <tr v-for="setting in filteredSettings" v-else
+                    :key="setting.key"
+                    :style="{'font-weight': (setting.modified) ? 'bold' : 'normal' }">
+                    <td><label :for="setting.key">{{ setting.key }}</label></td>
+                    <td v-if="setting.modified">
+                        <a class="u-link" @click="resetValue($event, setting.key)">
+                            {{ $t('settings_advanced_reset') }}
+                            <i class="fa fa-undo reset-icon" style="margin-left: 10px;"/>
+                        </a>
+                    </td>
+                    <td v-else />
+                    <td>
+                        <input v-if="setting.type === 'boolean'"
+                               :checked="setting.val"
+                               :id="setting.key"
+                               type="checkbox"
+                               @click="updateSetting($event, setting.key)">
+                        <input v-else-if="setting.type === 'number'"
+                               :value="setting.val"
+                               :id="setting.key"
+                               class="u-input"
+                               type="number"
+                               @keydown.13="$event.target.blur()"
+                               @change="updateSetting($event, setting.key)"
+                               @blur="updateSetting($event, setting.key)">
+                        <input v-else
+                               :value="setting.val"
+                               :id="setting.key"
+                               class="u-input"
+                               @keydown.13="$event.target.blur()"
+                               @blur="updateSetting($event, setting.key)">
+                    </td>
+                </tr>
+            </table>
+        </form>
+    </div>
+</template>
+
+<script>
+
+import state from '@/libs/state';
+import _ from 'lodash';
+
+export default {
+    data: function data() {
+        return {
+            filterString: '',
+            ignoreKeys: ['emojis', 'themes', 'bnc', 'aliases', 'restricted',
+                'hide_advanced', 'windowTitle', 'startupOptions', 'plugins'],
+        };
+    },
+    computed: {
+        filteredSettings() {
+            let settings = this.settings;
+            let out = [];
+            Object.values(settings).forEach((value, index) => {
+                if (value.key.toLowerCase().includes(this.filterString.toLowerCase())) {
```

Comment:
We can get a performance improvement and easier code reading if you move this filterString property outside of the loop. `let filter = this.filterString.toLowerCase();` then just using that filter var.